pyproject.toml

```diff
@@ -26,7 +26,8 @@ exclude = ["pyramid_openapi3/tests/"]
 [tool.poetry.dependencies]
 python = "^3.8"
 
-openapi-core = ">=0.16.1"
+# openapi-core = ">=0.16.1"
+openapi-core = { git = "https://github.com/zupo/openapi-core.git", branch = "patch-2" }
```

Comment:
See https://github.com/python-openapi/openapi-core/pull/544.

Reply:
My fixed was merged, I updated this PR back to using upstream. 

I'll pin to a specific version once a new release of openapi-core is cut.